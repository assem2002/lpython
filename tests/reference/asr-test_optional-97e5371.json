{
    "basename": "asr-test_optional-97e5371",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_optional.py",
    "infile_hash": "2532c73de47afb82c9d1d9fa335d3b274e7c79a14007353639958a45",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_optional-97e5371.stderr",
    "stderr_hash": "651a42d14e426b122a1048a63882de59e3508c8c68cdb1c25449e47a",
    "returncode": 2
}