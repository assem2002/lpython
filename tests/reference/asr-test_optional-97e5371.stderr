semantic error: The type 'Optional' is undeclared.
 --> tests/errors/test_optional.py:1:39
  |
1 | def match(pat : str, string : str) -> Optional[list[str] | str]:
  |                                       ^^^^^^^^^^^^^^^^^^^^^^^^^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
